src/lib/transaction_snark/transaction_snark.ml

```diff
@@ -4848,8 +4848,16 @@ module Make_str (A : Wire_types.Concrete) = struct
                     =
                   prover ~handler tx_statement
                 in
+                (* TODO: does this make sense? where did the hash come from? *)
+                let verification_key_hash =
+                  match simple_snapp_account_update.authorization with
```

Comment:
I think this needs to be passed to the function. Include it in `?zkapp_prover` ?

Reply:
Done.